host/sgx/windows/enter.asm

```diff
@@ -103,10 +111,11 @@ execute_eenter:
 
     mov ARG1OUT, rdi
     mov ARG2OUT, rsi
-	
-    ;; Restore the saved MXCSR
-    ldmxcsr MXCSR
-	
+
+    ;; Restore the saved host context
+    mov rcx, HOST_CONTEXT
+    call oe_restore_host_context
```

Comment:
Great work! Looks a lot cleaner.

Reading this new version, it looks like only the `execute_eenter` block needs to be in assembly.
The rest of this function can be written in C. Something we can attempt when there is bandwidth next release.

Reply:
I agree, we should consider factoring out more of this into higher level code instead of assembly in the next milestone.